recipes/newick-python/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "newick-python" %}
+{% set version = "0.8.0" %}
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  fn: newick-0.8.0.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 148371c1cce64093b3055e408f1b85576744ebc1085b9fa693e20f54d754b164
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - newick
+
+about:
+  home: 'https://github.com/glottobank/python-newick'
+  license: MIT License
```

Comment:
The license is Apache-2.0 acc. to
https://github.com/glottobank/python-newick/blob/v0.8.0/LICENSE
and
https://github.com/glottobank/python-newick/blob/v0.8.0/setup.py#L23.

I'd assume the entry `'License :: OSI Approved :: MIT License'` in
https://github.com/glottobank/python-newick/blob/v0.8.0/setup.py#L29
is wrong, probably as copy-paste error.